### PR TITLE
Remove dependency from X11::Protocol and use names instead of numbers for variable $type 

### PR DESCRIPTION
**Pull request by mimosinnet**
The script was not working as the variable "$type" was using strings instead of "numbers". Also, the dependency from X11::Protocol is no longer needed.

### CODE REVIEW

File: i3tree [hunk 1]
```diff
@@ -1,29 +1,32 @@
 #!/usr/bin/env perl
 
+# use {{{
 use strict;
 use warnings;
-
+use v5.20;
```
Comment (sciurus) [inline i3tree]: I'd like to support Perl back to 5.14; this is what ships with Debian Wheezy and EL7.

**Comment by sciurus** [comment]
Thanks for working on this!

Can you remove the commented-out code?

**Comment by mimosinnet** [comment]
Changed v5.20 to v.5.14 and removed the vim folding marks. Cheers!

**Comment by sciurus** [comment]
Looks good!

**Pull request merged**